hudi-common/src/main/java/org/apache/hudi/common/table/HoodieTableConfig.java

```diff
@@ -493,6 +544,13 @@ public String getUrlEncodePartitioning() {
     return getString(URL_ENCODE_PARTITIONING);
   }
 
+  /**
+   * Read the table checksum.
+   */
+  public Long getTableChecksum() {
```

Comment:
why public ?

Reply:
don't need it to be public, i'll make it private.